hw-latest/fpga/FPGA_Instructions.md

```diff
@@ -122,16 +120,41 @@ Serial port settings for connection over USB.
 1. Execute test targeting fpga_realtime
     - `CPTRA_UIO_NUM=4 cargo test -p caliptra-test --features=fpga_realtime smoke_test`
 
-### JTAG debug [Doc WIP]
-JTAG support has been enabled using OpenOCD. The security state debug_locked parameter must be FALSE for JTAG to be accessible.
-- `openocd.exe -f ~/openocd_caliptra.txt`
+### JTAG debug
+Requirements
+- Security state must have either debug_locked == false or lifecycle == manuf.
+- Set "debug = true" in firmware profile to provide line information to GDB.
+- Olimex ARM-USB-TINY-H with SiFive-Arty Adapter
 
-| JTAG Feature                               | Status              |
-| ------------------------------------------ | ------------------- |
-| RISCV register access                      | Works               |
-| VEER MSR                                   | Works               |
-| OpenOCD HW breakpoints                     | Works               |
-| SW Breakpoints                             | Requires System Bus |
-| RISCV Abstract access (ICCM and DCCM only) | Works               |
-| System Bus Access                          | Not connected       |
-| GDB                                        | Connects but hangs  |
\ No newline at end of file
+#### PMOD pin assignment ####
+![](./images/Caliptra_FPGA_PMOD_JTAG.svg)
+
+#### Debugger launch procedure ####
+1. Invoke OpenOCD server
+    - `openocd.exe -f ~/openocd_caliptra.txt`
+1. Connect client(s) for debug
+    - GDB: Port 3333
+    - Telnet: Port 4444
+
+#### Caliptra SoC interface registers ####
+Over Telnet connection to OpenOCD: `riscv.cpu riscv dmi_read [addr]`
+
+#### JTAG Testing ####
```

Comment:
lowercase `testing` in heading

Note: could do same throughout the doc (lowercase subsequent words in headings) but ok if out of scope for this PR.